Spanner/src/Transaction.php

```diff
@@ -704,4 +714,34 @@ private function enqueue($op, $table, array $dataSet)
             }
         }
     }
+
+    /**
+     * Build the update options.
+     *
+     * @param array $options The update options
+     * @return array
+     */
+    private function buildUpdateOptions(array $options): array
+    {
+        unset($options['requestOptions']['transactionTag']);
+        if (isset($this->tag)) {
+            $options += [
+                'requestOptions' => []
+            ];
```

Comment:
```suggestion
```

Reply:
We need to add the `requestOptions` key to the options if not present such that it can be used down the line. Any reason for this removal suggestion?

Reply:
Following line is             `$options['requestOptions']['transactionTag'] = $this->tag;` so this part seems redundant